<?xml version="1.0" encoding="UTF-8"?>

<!--
SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia <dev@dragon.best>

SPDX-License-Identifier: GPL-3.0-or-later
-->

<ui version="4.0">
 <class>LottieFilesSearchDialog</class>
 <widget class="QDialog" name="LottieFilesSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import From LottieFiles</string>
  </property>
  <property name="windowIcon">
   <iconset theme="lottiefiles">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="input_query"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset theme="search">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="result_area">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="result_area_content">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>785</width>
        <height>448</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="result_area_layout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar"/>
   </item>
   <item>
    <widget class="QLabel" name="label_error">
     <property name="text">
      <string notr="true">Network Error</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_previous">
       <property name="text">
        <string>Previous Page</string>
       </property>
       <property name="icon">
        <iconset theme="go-previous"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_next">
       <property name="text">
        <string>Next Page</string>
       </property>
       <property name="icon">
        <iconset theme="go-next">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_open">
       <property name="text">
        <string>Open</string>
       </property>
       <property name="icon">
        <iconset theme="document-open">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_import">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset theme="document-import">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_open</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>clicked_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>66</x>
     <y>656</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_import</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>clicked_import()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>614</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>781</x>
     <y>693</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>clicked_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>772</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>741</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_next</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>clicked_next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>723</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_previous</sender>
   <signal>clicked()</signal>
   <receiver>LottieFilesSearchDialog</receiver>
   <slot>clicked_previous()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>724</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clicked_open()</slot>
  <slot>clicked_import()</slot>
  <slot>clicked_search()</slot>
  <slot>clicked_next()</slot>
  <slot>page_changed(int)</slot>
  <slot>clicked_previous()</slot>
 </slots>
</ui>
